src/public/js/video-tagging/video-tagging.html

```diff
@@ -954,10 +963,13 @@
     initImageDir: function() {
         if (!this.imagelist) return;
```

Comment:
for consistency/readability, I'd structure conditionals like: 

```
if (blah) {

} 
```
Pretty sure this is in line w/ most linters, but I'd have to double check...